### ISSUE TITLE
Postbacks blocking code if postback server times out

### ISSUE THREAD

**Issue by zoton2** [esamarathon/donation-tracker#6]
If there are any postback URLs set in the settings, and the server they are pointing at doesn't respond in 5 seconds, the server returns a 500(?) error and the rest of the code seems to be ignored.

The code does have that timeout parameter (`opener.open(req, timeout=5)`), the timeout should just fail gracefully, or we should change to using something that is async so it doesn't actually block the code if it's waiting.

**Comment by zoton2** [comment]
And now I post that, it seems to be a different matter. If the postback server doesn't exist (so I assume a 404) it will be fine, but if the postback server does exist but it doesn't return anything, it will block the code.

**Comment by zoton2** [comment]
No, from some more testing, it seems like it *does* happen if the server doesn't exist too.

**Comment by zephyyrr** [comment]
I guess you just have to make sure the server is always there... listening...
